logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/GlobalCallManager.kt

```diff
@@ -7,6 +7,7 @@ import com.wire.kalium.logic.data.user.UserRepository
 expect class GlobalCallManager {
 
     fun getCallManagerForClient(
+        userId: String,
```

Comment:
It's a UserId not a String 🙂
As we are passing the userId, do we need the userRepository ?

Reply:
It is actually a `NonQualifiedUserId` from `network.api` package, but I have already changed it from String.

We are passing `UserId` as `NonQualifiedUserId` to use it as an initial key for holding the `CallManager` so we don't have it created multiple times for one single user.

The `userRepository` we are using to get the correct `UserId` with `QualifiedID`, thus the necessity to have both as they are exercising different things later on.

TLDR: NonQualifiedUserId for holding CallManager instances, userRepository to get user QualifiedID.